sabotage_field/sabotage_dice_result.gd

```diff
@@ -0,0 +1,83 @@
+extends CanvasLayer
+
+@onready var label=$CanvasLayer/Panel/Label
+
+signal finished
+
+var player: Player
+var action: Callable
+
+
+func _on_button_pressed():
+	await action.call()
+	finished.emit()
+
+
+func display(player: Player, dice_value):
+	self.player = player
+	visible = true
+
+	var duel_50 = {
+		"text": "Duell Einsatz: 50 Riesen",
+		"action": func():
+			var target = await get_target()
+			var result = await run_duel(target)
+			# match case over result
+	}
+
+
+	if (dice_value==7 || dice_value==8):
+		result="Big Baller Duell Einsatz: halbes Vermögen"
```

Comment:
The variable `result` is not declared in the current scope, seems to be a recent change that broke it.